tests/backup/backup_resiliency_test.go

```diff
@@ -3,20 +3,22 @@ package tests
 import (
 	"context"
 	"fmt"
-	"strings"
```

Comment:
What's the point of removing it and moving it down?

Reply:
done